### PR TITLE
Issue 55 remove refinements

### PR DESCRIPTION
**Pull request by senier**
This proves now with >= https://github.com/Componolit/RecordFlux/commit/1a4eff261d1baa3e35e3299f200ba7b2e332f8ec, verification takes ~15min though.

### CODE REVIEW

File: spdm.rflx [hunk 1]
```diff
@@ -1129,28 +1048,144 @@ package SPDM is
...
+         Challenge_Request : Challenge_Request
+            then null;
+         Respond_If_Ready_Request : Respond_If_Ready_Request
+            then null;
+         Key_Exchange_Request : Key_Exchange_Request
+            then null;
+         Finish_Request : Finish_Request
+            then null;
+         End_Session_Request : End_Session_Request;
+      end message;
```
Comment (jklmnn) [inline spdm.rflx:1123]: This is incomplete, e.g. Get_Measurements is missing completely.

File: spdm.rflx [hunk 2]
```diff
@@ -1129,28 +1048,144 @@ package SPDM is
...
+         Error_Response_1_0 : Error_Response_1_0
+            then null;
+         Error_Response_1_1 : Error_Response_1_1
+            then null;
+         Key_Exchange_Response : Key_Exchange_Response
+            then null;
+         Finish_Response : Finish_Response
+            then null;
+         End_Session_Response : End_Session_Response;
+      end message;
```
Comment (jklmnn) [inline spdm.rflx:1189]: Same as for Request, e.g. `Measurements` is missing completely.

**Review by jklmnn — CHANGES_REQUESTED** [review]

File: spdm.rflx [hunk 3]
```diff
@@ -1129,28 +1032,144 @@ package SPDM is
...
+            then Challenge_Auth_Response_1_1
+               if Code = Challenge_Auth and Minor_Version = 1
+            then Error_Response_1_0
+               if Code = Error and Minor_Version = 0
+            then Error_Response_1_1
+               if Code = Error and Minor_Version = 1
+            then Key_Exchange_Response
+               if Code = Key_Exchange_RSP and Minor_Version = 1
+            then Finish_Response
+               if Code = Finish_RSP
```
Comment (jklmnn) [inline spdm.rflx:1143]: This message also requires SPDM 1.1.

**Review by jklmnn — CHANGES_REQUESTED** [review]

File: spdm.rflx (hunk 3 above)
Comment (senier) [reply spdm.rflx]: Changed in 40b2073d86b53873a66e7160db66b7f90f15b435

File: spdm.rflx [hunk 4]
```diff
@@ -820,21 +759,15 @@ package SPDM is
          --    - contain a hash if Hash_Type was TCB_Measurement
          --    - be 0 if Hash_Type was TCB_Measurement and no measurement is required
          --    - contain a hash of all measurements if Hash_Type was All_Measurments
-         Measurement_Summary_Hash : Hash;
```
[the remark this answers is not in the document]
Comment (senier) [reply spdm.rflx]: Merge error fixed in ae007efd06dd5b1429aa02c493e21e04087afb65

File: spdm.rflx [hunk 5]
```diff
@@ -1025,15 +951,13 @@ package SPDM is
             -- ISSUE: Componolit/RecordFlux#609
             -- Size of ExchangeData is determined by the algorithm selected in previous ALGORITHMS
             -- message
-            with Size => 8 * 96;
```
[the remark this answers is not in the document]
Comment (senier) [reply spdm.rflx]: Merge error fixed in ae007efd06dd5b1429aa02c493e21e04087afb65

File: spdm.rflx [hunk 6]
```diff
@@ -1070,10 +994,11 @@ package SPDM is
             -- ISSUE: Componolit/RecordFlux#609
             -- Size of MeasurementSummaryHash is determined by the algorithm selected in previous
             -- ALGORITHMS message
-            with Size => 8 * 76;
```
[the remark this answers is not in the document]
Comment (senier) [reply spdm.rflx]: Merge error fixed in ae007efd06dd5b1429aa02c493e21e04087afb65

**Review by jklmnn — APPROVED** [review]

**Pull request merged**